apps/ledger-live-mobile/src/logic/postOnboarding/index.tsx

```diff
@@ -0,0 +1,122 @@
+import { DeviceModelId, getDeviceModel } from "@ledgerhq/devices/lib/index";
+import {
+  PostOnboardingAction,
+  PostOnboardingActionId,
+} from "@ledgerhq/types-live/lib/postOnboarding";
+import { Icons } from "@ledgerhq/native-ui";
+import { NavigatorName, ScreenName } from "../../const";
+
+const claimMock: PostOnboardingAction = {
+  id: PostOnboardingActionId.claimMock,
+  icon: Icons.GiftCardMedium,
+  title: "Claim my NFT",
+  description: "A special NFT for you.",
+  tagLabel: "Free",
+  actionCompletedPopupLabel: "NFT claimed",
+  actionCompletedHubTitle: "Kickstart your Web3 journey.",
+  navigationParams: [
+    NavigatorName.PostOnboarding,
+    {
+      screen: ScreenName.PostOnboardingMockActionScreen,
+      params: {
+        id: PostOnboardingActionId.claimMock,
+        title: PostOnboardingActionId.claimMock,
+      },
+    },
+  ],
+};
+
+const personalizeMock: PostOnboardingAction = {
+  id: PostOnboardingActionId.personalizeMock,
+  icon: Icons.BracketsMedium,
+  featureFlagId: "customImage",
+  title: `Personalize my ${getDeviceModel(DeviceModelId.nanoFTS).productName}`,
+  description: "By customizing the screen.",
+  actionCompletedPopupLabel: "Device personalized",
+  actionCompletedHubTitle: "That screen is looking neat.",
+  navigationParams: [
+    NavigatorName.PostOnboarding,
+    {
+      screen: ScreenName.PostOnboardingMockActionScreen,
+      params: {
+        id: PostOnboardingActionId.personalizeMock,
+        title: PostOnboardingActionId.personalizeMock,
+      },
+    },
+  ],
+};
+
+const migrateAssetsMock: PostOnboardingAction = {
+  id: PostOnboardingActionId.migrateAssetsMock,
+  icon: Icons.TransferMedium,
+  title: "Transfer assets to my Ledger",
+  description: "Easily secure assets from coinbase or another exchange.",
+  actionCompletedPopupLabel: "Assets transfered",
+  actionCompletedHubTitle: "Something about being a crypto pro.",
+  navigationParams: [
+    NavigatorName.PostOnboarding,
+    {
+      screen: ScreenName.PostOnboardingMockActionScreen,
+      params: {
+        id: PostOnboardingActionId.migrateAssetsMock,
+        title: PostOnboardingActionId.migrateAssetsMock,
+      },
+    },
+  ],
+};
+
+/**
+ * All implemented post onboarding actions.
+ */
+const postOnboardingActions: Record<
+  PostOnboardingActionId,
+  PostOnboardingAction
+> = {
+  claimMock,
+  migrateAssetsMock,
+  personalizeMock,
+};
+
+/**
+ * Mock of post onboarding actions for DeviceModelId.nanoFTS
+ */
+const ftsPostOnboardingActionsMock: PostOnboardingAction[] = [
+  claimMock,
+  personalizeMock,
+  migrateAssetsMock,
+];
+
+export function getPostOnboardingAction(
+  id: PostOnboardingActionId,
+): PostOnboardingAction {
+  return postOnboardingActions[id];
+}
+
+/**
+ * Returns the list of post onboarding actions for a given device.
+ */
+export function getPostOnboardingActionsForDevice(
+  deviceModelId: DeviceModelId,
+  mock = false,
+): PostOnboardingAction[] {
+  switch (deviceModelId) {
```

Comment:
Since this seems to be a placeholder for the real thing, I suggest adding a TODO or FIXME so that it's easier to find again. But if you are sure you will not forget about it... :trollface:

Reply:
I'll add a TODO but I'm not sure it will help much haha.

There is no real thing for now 🤷
None of the real actions are implemented. They will have to be added to the post onboarding as the features get implemented (we will add that in the grooming of these features and for the ones already started I'm going to add a subtask on Jira).

I documented the process of adding a new action here for that purpose: 
https://github.com/LedgerHQ/ledger-live/wiki/LLM:PostOnboarding